src/components/Note/Note.jsx

```diff
@@ -18,7 +18,10 @@ function Note({ note }) {
       <label>
         Category:
         <select>
-          <option>{note.category}</option>
+          <option value={note.category}>{note.category}</option>
```

Comment:
Eventually this could be shortened to a defined array of options for this field, and then `map()` over them to create all the options.

Like:
```
const categories = ['Bourbon', 'Rye', 'American Single Malt']
```

Let's just delete the 3 hard-coded options for now, and we can add them a different way later when we get text inputs all wired up.